ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-community-4.12-periodics.yaml

```diff
@@ -20,10 +20,11 @@ periodics:
       job_states_to_report:
       - failure
       - error
-      report_template: '<!subteam^S025TMX0LF3> :warning: Job *{{.Spec.Job}}* ended
-        with *{{.Status.State}}*. Commit reference: *{{.Refs.Baselink}}* <{{.Status.URL}}|View
-        logs>. Please investigate. :warning: Recommended not to use this bundle for
-        the next community WMCO release.'
+      report_template: '{{if eq .Status.State "success"}} <!subteam^S025TMX0LF3> :white_check_mark:
```

Comment:
IMO we don't need to tag the winc-watcher on success.

Reply:
I agree - this is just temporary to make sure we're getting alerts at all. I'm going to have to open another followup once we're sure its working

Reply:
Cool, lgtm in that case